src/api/subscriptions.js

```diff
@@ -221,8 +221,11 @@ export let LoginRequest;
 
 
 /**
+ * Properties:
+ * - one and only one of "token" or "authCode"
```

Comment:
Let's document these two links here: for authCode: https://developers.google.com/actions/identity/oauth2-code-flow
and for tokenhttps://developers.google.com/actions/identity/oauth2-implicit-flow

Reply:
Done.